console/src/assets/i18n/de.json

```diff
@@ -881,7 +881,8 @@
       "BRANDING": "Branding",
       "PRIVACYPOLICY": "Datenschutzrichtlinie",
       "OIDC": "OIDC Token Lifetime und Expiration",
-      "SECRETS": "Secret Erscheinungsbild"
+      "SECRETS": "Secret Erscheinungsbild",
```

Comment:
It this on purpose half in english?

Reply:
i checked the current translation, we always have "secret" as in client secret, ...